<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicineView</class>
 <widget class="QWidget" name="MedicineView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>158</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="actionImportCSV">
      <property name="text">
       <string>批量导入</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="actionExportCSV">
      <property name="text">
       <string>批量导出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>641</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="placeholderText">
         <string>请输入姓名进行查找</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnFind">
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnAdd">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnDelete">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnChange">
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="myTableView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
